procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb

```diff
@@ -5,10 +5,10 @@
    "id": "3dbf3c0c",
```

Comment:
<p>So this should really be using the enumerations.</p><p>see:</p><p>https://github.com/lsst-ts/ts_idl/blob/develop/python/lsst/ts/idl/enums/ATMCS.py#L34</p><p> </p><pre class="ql-syntax" spellcheck="false">from lsst.ts.observatory.control.utils import M3ExitPort
await atcs.rem.atmcs.cmd_setInstrumentPort.set_start(port=M3ExitPort.Nasmyth1)

</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/92/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

Reply:
Changed. Thank you for the input and teaching.